packages/synthchron-tauri/src/components/react-flow/processModels/processModelFlowConfig.ts

```diff
@@ -1,11 +1,14 @@
-import { EdgeTypes, NodeTypes, Node, Connection } from 'reactflow'
+import { PetriNetProcessModel } from '@synthchron/simulator'
+import { EdgeTypes, NodeTypes, Node, Edge, Connection } from 'reactflow'
 
 export type ProcessModelFlowConfig = {
+  processModelType: string
   nodeTypes: NodeTypes
   edgeTypes: EdgeTypes
   checkConnect: (
     connection: Connection,
     sourceNode: Node,
     targetNode: Node
   ) => Connection | null
+  serialize: (nodes: Node[], edges: Edge[]) => PetriNetProcessModel
```

Comment:
This is incorrect, in general the serialize method returns any process model, not just a petri net process model